src/main/view.ts

```diff
@@ -51,6 +52,14 @@ export class View extends BrowserView {
     this.window = window;
     this.homeUrl = url;
 
+    const { object: settings } = windowsManager.settings;
+    if (settings.doNotTrack) {
```

Comment:
The `if` could be moved into the handler, so that it always updates when the setting changes.

Reply:
> The `if` could be moved into the handler, so that it always updates when the setting changes.

which handler do you mean?

Reply:
Inside the `onBeforeSendHeaders` handler.